app/controllers/evaluations_controller.rb

```diff
@@ -50,38 +55,61 @@ def new
   end
 
   def edit
-    @evaluation = Evaluation.find_by(id: params[:id])
+    @evaluation = Evaluation.includes([evaluation_scores: :evaluation_criterion]).find_by(id: params[:id])
     @evaluator_submission_assignment = find_evaluator_submission_assignment
     return unauthorized_redirect unless can_access_evaluation?
 
     render :edit
   end
 
-  def save_draft
-    @evaluation.completed_at = nil
+  def create
+    if save_evaluation
+      flash[:notice] =
+        if params[:subaction] == "mark_complete"
+          I18n.t("evaluations.notices.marked_complete")
+        else
+          I18n.t("evaluations.notices.saved_draft")
+        end
 
-    if @evaluation.save(validate: false)
-      flash[:notice] = I18n.t("evaluations.notices.saved_draft")
-      redirect_to evaluations_path
+      redirect_to submissions_evaluation_path(@evaluation.submission)
```

Comment:
![Screenshot 2025-01-30 at 7 58 38 PM](https://github.com/user-attachments/assets/9e550f99-8280-4660-9285-8f26bb8d0ff0)

I get an error page when I Save Draft. The evaluation draft is saved, but this is redirecting to the incorrect url. I think it should be the phase id here...
```suggestion
      redirect_to submissions_evaluation_path(@evaluation.submission.phase)
```